src/main/java/com/regnosys/testing/pipeline/PipelineModelBuilder.java

```diff
@@ -57,12 +58,14 @@ protected PipelineModel build(PipelineNode modelBuilder, PipelineTreeConfig conf
         String pipelineId = modelBuilder.id(config.isStrictUniqueIds());
         String upstreamPipelineId = modelBuilder.upstreamId(config.isStrictUniqueIds());
 
+        String prefixedPipelineName = String.format("%s %s", config.getPrefix(), name);
         return new PipelineModel(pipelineId,
-                name,
+                prefixedPipelineName,
                 new PipelineModel.Transform(modelBuilder.getTransformType(), modelBuilder.getFunction().getName(), inputType, outputType),
                 upstreamPipelineId,
                 inputSerialisation,
-                outputSerialisation);
+                outputSerialisation,
+                config.getPrefix());
```

Comment:
I find this a little bit confusing. The `prefix` is used as the model name here. Is the prefix always equal to the name of the model? If so, why not call it `modelName`? Less mental mapping - much more readable.